tardis/montecarlo/base.py

```diff
@@ -228,10 +228,10 @@ def _initialize_packets(
             )
 
         mc_config_module.packet_seeds = seeds
-        self.input_r = radii
-        self.input_nu = nus
+        self.input_r = radii.to(u.cm)
```

Comment:
Should this have a `.value`?

Reply:
@smithis7 
No, radii was already coded in a unit sensitive manner, so no changes were needed here. I just decided to add that "to" function as a sanity check (refactoring), because otherwise the codebase assumes that quantities would be given in CGS. (The test cases pass without that "to" function too)

Also, "mus" is a dimensionless quantity, so nothing to be done there.

Reply:
@smithis7 I have also cleared Jupyter cell outputs and ran the documentation build. Just a minor change was needed in 1 other notebook, fixed it. The documentation build was successful.

Please review